carma_wm/src/CARMAWorldModel.cpp

```diff
@@ -263,10 +263,21 @@ std::vector<lanelet::ConstLanelet> CARMAWorldModel::getLaneletsBetween(double st
     TrackPos min = routeTrackPos(front);
     TrackPos max = routeTrackPos(back);
 
-    if (std::max(min.downtrack, start) > std::min(max.downtrack, end))
-    {  // Check for 1d intersection
-      // No intersection so continue
-      continue;
+    if (!bounds_inclusive) // reduce bounds slightly to avoid including exact bounds
+    {
+      if (std::max(min.downtrack, start + 0.1) > std::min(max.downtrack, end - 0.1))
```

Comment:
Looks like you might have clobbered you previous fix for the bounds size here and additional comment describing this assumption.

Reply:
added back the smaller size and comments in the header